tutorials/nlp/Question_Answering_Squad.ipynb

```diff
@@ -46,7 +46,7 @@
         "id": "uRLPr0TnIAHO"
       },
       "source": [
-        "BRANCH = 'v1.0.0b3'"
+        "BRANCH = 'main'"
```

Comment:
I am not sure what is the NeMo policy for the notebooks now... @okuchaiev ?

Reply:
this change will not work in the Notebook if this is not changed

Reply:
looks good to me, see Evelina's PR on notebook updates

Reply:
I'd say it's ok to have an exception like this. But just keep in mind that this makes this notebook somewhat vulnerable because it is loaded from main